pkg/cloudfoundry/containertagger/container_tagger.go

```diff
@@ -87,10 +87,12 @@ func (c *ContainerTagger) Start(ctx context.Context) {
 }
 
 func (c *ContainerTagger) processEvent(ctx context.Context, evt workloadmeta.Event) error {
-	containerID := evt.Entity.GetID().ID
-
+	entity := evt.Entity
+	containerID := entity.GetID().ID
 	if evt.Type == workloadmeta.EventTypeSet {
-		storeContainer := evt.Entity.(*workloadmeta.Container)
+		eventTimestamp := time.Now().UnixNano()
+		storeContainer := entity.(*workloadmeta.Container)
+		log.Debugf("Processing Event (id %d): %+v", eventTimestamp, storeContainer)
```

Comment:
Same here, let's create a `eventID := fmt.Sprintf("%s-%d", containerId, eventTimestamp)` and use it everywhere.